engine/lib/pagehandler.php

```diff
@@ -244,6 +244,13 @@ function elgg_front_page_handler() {
  * @return void
  */
 function elgg_error_page_handler($hook, $type, $result, $params) {
+	static $once = 0;
+	$once++;
+	if ($once === 2) {
+		echo "Infinite loop attempting to show error page.";
```

Comment:
This happens if the resources/error view can't output anything. It could mean the locations cache was corrupt, but basically this shouldn't happen in a stable release, so we could probably remove this. It was just making me nuts while I was debugging.